benches/benches.rs

```diff
@@ -93,7 +93,7 @@ fn http3(c: &mut Criterion) {
 
     let mut config = make_bench_config();
 
-    let h3_config = quiche::h3::Config::new().unwrap();
+    let mut h3_config = quiche::h3::Config::new().unwrap();
```

Comment:
Seems `mut` is unnecessary? (then you need to change L99 too)

Reply:
No, this file currently doesn't build, so this fixes that.